Source/SwiftLintBuiltInRules/Rules/Lint/QuickDiscouragedCallRule.swift

```diff
@@ -47,6 +47,7 @@ struct QuickDiscouragedCallRule: OptInRule {
         guard
             kind == .call,
             let name = dictionary.name,
+            name != "@TestState", // İstisna: @TestState için uyarı tetiklenmesin
```

Comment:
Please use English for comments.

I don't think this fix is correct. `name` will never be `@TestState`. These are the names of Quick methods, not variables or attributes.